src/coreclr/inc/corinfoinstructionset.h

```diff
@@ -14,7 +14,7 @@
 enum CORINFO_InstructionSet
 {
     InstructionSet_ILLEGAL = 0,
-    InstructionSet_NONE = 63,
+    InstructionSet_NONE = 41,
```

Comment:
nit: I think `NONE` needs to stay as the last in-use bit (so `(FlagsArrSize * sizeof(uint64_t) * 8) - 1`)

Reply:
In the generator code(which is c#), we are hardcoding  `sizeof(uint64_t)` as 64 to calculate `FlagsArrSize` and `sizeof(uint64_t)` as such obviously will not work in the cs code(`InstructionsetGenerator.cs`). As such, you think `(FlagsArrSize * 64)  - 1` works here?

Reply:
That works too.

Reply:
Changed